src/styles/components/_navbar.scss

```diff
@@ -29,4 +21,44 @@
   @include respond-above('xl') {
     padding: 0 rem(40px);
   }
+
+  &__menu-item,
+  &__authentication-button {
+    &:not(:last-child) {
+      margin-right: rem(16px);
+    }
+  }
+
+  &__logo {
+    display: flex;
+  }
+
+  &__menu {
+    list-style-type: none;
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
+
+  // TODO: for future implementation
+  &__menu-item {
+  }
+
+  &__link {
+    text-decoration: none;
+  }
+
+  &__account {
+    display: flex;
```

Comment:
Just `display: flex` does nothing. Are you going to add any alignments or others?

Reply:
Oh this is typo 😅